setup.py

```diff
@@ -5,10 +5,6 @@
 
 here = path.abspath(path.dirname(__file__))
 
-# Get the long description from the relevant file
```

Comment:
removed reading of the long description from the file as rst file is not maintained and removed (above). This can be added later, if required.